### PR TITLE
Fix future snapshots

### PR DESCRIPTION
**Pull request by ChaituVR**

### CODE REVIEW

File: src/rpc.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+  blockNumByNetwork
+} from './utils';
 import { version } from '../package.json';
 
 const router = express.Router();
 
 router.post('/', async (req, res) => {
   const { id = null, params = {} } = req.body;
   try {
+    typeof params.snapshot != 'number' && (params.snapshot = 'latest');
```
Comment (bonustrack) [inline src/rpc.ts:21]: What's this?

Comment (ChaituVR) [reply src/rpc.ts:21]: If anyone passes something other than a block number, we now use `'latest'`. we don't need it?

Comment (bonustrack) [reply src/rpc.ts:21]: This is hard to read I would rather do
```
if (typeof params.snapshot !== 'number') params.snapshot = 'latest';
```

File: src/scores.ts [hunk 2]
```diff
@@ -37,10 +20,10 @@ async function calculateScores(parent, args, key) {
     parent.requestId
   );
 
-  let snapshotBlockNum = 'latest';
-  if (args.snapshot !== 'latest') {
```
Comment (bonustrack) [inline src/scores.ts:41]: This code is working fine, any reason to change it? With longer code

Comment (ChaituVR) [reply src/scores.ts:41]: Hey yes,, trying to do the same thing as in `getVp` to make it work with `latest` if we pass any unknown parameter into the snapshot param

**Review by bonustrack — APPROVED** [review]

**Pull request merged**